src/nav-soknad/utils/amplitude.ts

```diff
@@ -0,0 +1,30 @@
+const amplitude = typeof window !== "undefined" ? require("amplitude-js") : () => null;
+
+export const initAmplitude = () => {
+    if (amplitude) {
+        amplitude.getInstance().init("default", "", {
+            apiEndpoint: "amplitude.nav.no/collect-auto",
+            saveEvents: false,
+            includeUtm: true,
+            includeReferrer: true,
+            platform: window.location.toString(),
+        });
+    }
+};
+
+export function logAmplitudeEvent(eventName: string, eventData?: Record<string, unknown>): void {
+    setTimeout(() => {
+        try {
+            if (amplitude) {
+                amplitude.getInstance().logEvent(eventName, eventData);
+            }
+        } catch (error) {
+            console.error(error);
+        }
+    });
+}
+
+export const createSkjemaEventData = () => ({
+    skjemaId: "sosialhjelpsoknad",
+    skjemanavn: "Soknad om økonomisk sosialhjelp",
```

Comment:
kunne dette vært lagt til på eventData inni logAmplitudeEvent (hvis den alltid er lik)? Så slipper man å kalle funksjonen i hver logging :)

Reply:
Tanken var å få lagt på logging for ettersendelser etterhvert også, og da vet jeg ikke om skjemaId/skjemanavn er like relevant?

Reply:
👍